app/views/404.hbs

```diff
@@ -0,0 +1,66 @@
+    {{! This incurs no performance hit. The snippet creates an asynchronous (non-blocking) request. }}
+    {{! This snippet has been edited to force analytics.js to load over https }}
+    <script nonce="{{../nonce.google_analytics}}">
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+      ga('create', '{{.}}', 'auto');
+      ga('send', 'pageview');
+    </script>
+    {{/config.google_analytics_account}}
+
+    {{#config.mixpanel_token}}
+    <script nonce="{{../nonce.mixpanel}}">
+      (function(e,a){if(!a.__SV){var b=window;try{var c,l,i,j=b.location,g=j.hash;c=function(a,b){return(l=a.match(RegExp(b+"=([^&]*)")))?l[1]:null};g&&c(g,"state")&&(i=JSON.parse(decodeURIComponent(c(g,"state"))),"mpeditor"===i.action&&(b.sessionStorage.setItem("_mpcehash",g),history.replaceState(i.desiredHash||"",e.title,j.pathname+j.search)))}catch(m){}var k,h;window.mixpanel=a;a._i=[];a.init=function(b,c,f){function e(b,a){var c=a.split(".");2==c.length&&(b=b[c[0]],a=c[1]);b[a]=function(){b.push([a].concat(Array.prototype.slice.call(arguments,
+      0)))}}var d=a;"undefined"!==typeof f?d=a[f]=[]:f="mixpanel";d.people=d.people||[];d.toString=function(b){var a="mixpanel";"mixpanel"!==f&&(a+="."+f);b||(a+=" (stub)");return a};d.people.toString=function(){return d.toString(1)+".people (stub)"};k="disable time_event track track_pageview track_links track_forms register register_once alias unregister identify name_tag set_config reset people.set people.set_once people.increment people.append people.union people.track_charge people.clear_charges people.delete_user".split(" ");
+      for(h=0;h<k.length;h++)e(d,k[h]);a._i.push([b,c,f])};a.__SV=1.2;b=e.createElement("script");b.type="text/javascript";b.async=!0;b.src="undefined"!==typeof MIXPANEL_CUSTOM_LIB_URL?MIXPANEL_CUSTOM_LIB_URL:"file:"===e.location.protocol&&"//cdn.mxpnl.com/libs/mixpanel-2-latest.min.js".match(/^\/\//)?"https://cdn.mxpnl.com/libs/mixpanel-2-latest.min.js":"//cdn.mxpnl.com/libs/mixpanel-2-latest.min.js";c=e.getElementsByTagName("script")[0];c.parentNode.insertBefore(b,c)}})(document,window.mixpanel||[]);
+      mixpanel.init("{{.}}");
+    </script>
+    {{/config.mixpanel_token}}
+
+  </head>
+  <body tabindex="1">
+
+    {{! SVG staging area }}
+    <div id="svg"></div>
+
+    {{! Mount point for React }}
+    <div id="react-app" class="app">
+      <div id="error">
+        <div class="clouds-background">
+          <div class="rear-clouds"></div>
+          <div class="front-clouds"></div>
+        </div>
+        <div class="error-content">
+          <h1><span>Page not found.</span></h1>
+          <div>
+            <span>Oh, boy. There is no page with the address</span><br>
```

Comment:
The text here should read: `Oh, boy. There is no page with this address!` Make sure the spelling and punctuation match.